webpack.config.js

```diff
@@ -10,21 +10,30 @@ const {version} = require('./packages/node_modules/webex/package.json');
 dotenv.config();
 dotenv.config({path: '.env.default'});
 
-module.exports = (env = process.env.NODE_ENV || 'production', args) => ({
-  entry: env === 'development' ?
+/**
+ * Webpack Config
+ *
+ * @param {object} [env]
+ * @param {string} env.NODE_ENV
+ * @returns {object}
+ */
+module.exports = (env = {NODE_ENV: process.env.NODE_ENV || 'production'}) => ({
+  entry: env && env.NODE_ENV === 'development' ?
     `${path.resolve(__dirname)}/packages/node_modules/webex/src/index.js` :
     './packages/node_modules/webex',
-  mode: env === 'development' ? 'development' : 'production',
+  mode: env && env.NODE_ENV === 'development' ? 'development' : 'production',
   output: {
     filename: 'webex.min.js',
     library: 'Webex',
     libraryTarget: 'umd',
     sourceMapFilename: '[file].map',
-    path: args && args.env && args.env.umd ? // samples:test fix since its called as a function
+    path: env && env.umd ? // `samples:test` script calls Webpack config as a function so `args` won't exist
```

Comment:
Seems like you got rid of `args` altogether? In that case the comment is not necessary
```suggestion
    path: env && env.umd ?
```